src/modules/Users.pm

```diff
@@ -4483,7 +4483,7 @@ sub Write {
 
 		my %user	        = %{$modified_users{$type}{$username}};
 		my $home 	        = $user{"homeDirectory"} || "";
-		my $use_btrfs_subvolume = $user{"btrfs_subvolume"} || 0;
```

Comment:
Note for reviewers: the `bool` subroutine [already returns `0`](https://github.com/yast/yast-users/blob/f71d13b8acaa77f90c3c4ff6b7d1454eef85362f/src/modules/Users.pm#L301-L303) if `$user{"btrfs_subvolume"}` is not defined.